test/UsersController.test.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+import type { TokenObject } from '../src/types';
+
+import test from 'ava';
+import request from 'supertest-as-promised';
+import ouathClients from '../src/oauthClients.json';
+
+import { app } from '../src/main';
+import settings from '../src/settings';
+
+const USER_CREDENTIALS = {
+  password: 'password',
+  username: 'newUser@test.com',
+};
+
+let user;
+let userToken;
+
+test.serial('should return a new user object', async t => {
+  const response = await request(app)
+    .post('/v1/users')
+    .send(USER_CREDENTIALS);
+
+  user = response.body;
+
+  t.is(response.status, 200);
+  t.truthy(user.username === USER_CREDENTIALS.username);
+  t.truthy(user.id && user.passwordHash && user.salt && user.created_at);
```

Comment:
not nessecary, it works okey in one line, too :
![assert](https://cloud.githubusercontent.com/assets/8849922/20908832/5e21c408-bb60-11e6-9e25-90c839db5c13.png)